src/sql/src/plan/statement/dml.rs

```diff
@@ -329,7 +329,7 @@ pub fn plan_explain(
         desc,
         finishing,
         scope: _,
-    } = query::plan_root_query(scx, query, QueryLifetime::OneShot)?;
+    } = query::plan_root_query(scx, query, QueryLifetime::OneShot)?; // TODO: Set correct QueryLifetime
```

Comment:
This is fixed in #21383 because the block now only covers the `EXPLAIN SELECT ...` case.